src/utils/function-utils.ts

```diff
@@ -48,4 +48,13 @@ export function convertToArray<T>(input: T | Array<T>): Array<T> {
   return isArray(input) ? input : [input];
 }
 
+/**
+ * Checks if React component is wrapped with forwardRef function
+ * @param elementType
+ */
+export const isForwardedRef = (elementType: ElementType): elementType is ForwardRefExoticComponent<unknown> => {
```

Comment:
dangerous - what happens if React decides to change this Symbol. 

Your users have multiple versions of React, is this true for all React versions

Reply:
@YossiSaadi you have any ideas how to make it better maybe?

Reply:
@SergeyRoyt Let's consider using [react-is](https://www.npmjs.com/package/react-is).
The numbers talk for themselves, this lib is awesome.
```
import { isForwardRef: reactIsIsForwardRef } from 'react-is';

function isForwardRef(element: React.ReactElement): boolean {
    return reactIsIsForwardRef(element);
}
```
or something similar
I think `React.ReactElement` should do the job from TS side.

Reply:
Removed - using `react-is` instead